src/main/java/org/openrewrite/java/testing/cleanup/AssertEqualsNullToAssertNull.java

```diff
@@ -54,7 +54,7 @@ protected JavaVisitor<ExecutionContext> getVisitor() {
             private Supplier<JavaParser> javaParser(ExecutionContext ctx) {
                 if(javaParser == null) {
                     javaParser = () -> JavaParser.fromJavaVersion()
-                            .classpathFromResources(ctx, "junit-jupiter-api-5.9.2")
+                            .classpathFromResources(ctx, "junit-jupiter-api-5.9.+")
```

Comment:
I am unsure about this. Looking at the implementation it looks like the string here will be interpreted as the prefix in a regex, where both `.` and `+` have a special meaning. I still don't understand why the builds fail in CI but not locally.

Reply:
Hmm; yes we seem to be inconsistent on the main branch when you compare the search results:
https://github.com/search?q=repo%3Aopenrewrite%2Frewrite-testing-frameworks%205.9.2&type=code
https://github.com/search?q=repo%3Aopenrewrite%2Frewrite-testing-frameworks+5.9.%2B&type=code

I picked `5.9.+` as it was already present often, and to pick up the new 5.9.3 that was released three weeks ago.

We can standardize on something else in this PR if that's preferred, although `.+` without escaped dots is pretty common throughout our repositories already. I'd assumed that to be an [internal version selector](https://docs.openrewrite.org/reference/dependency-version-selectors), but I guess it's indeed a regex as we don't support `.+`, only `.x` and others according to the docs.